pom.xml

```diff
@@ -1102,6 +1103,18 @@
                 <type>test-jar</type>
             </dependency>
 
+            <dependency>
+                <groupId>io.trino</groupId>
+                <artifactId>trino-filesystem-alluxio</artifactId>
+                <version>${project.version}</version>
+                <exclusions>
```

Comment:
This should be excluded from the dependency that pulls it in

Reply:
this is from grpc-netty so remove exclusion

Reply:
done